docs/modules/ZSPySDK/examples/zs_api_sdk.py

```
@@ -212,10 +213,11 @@ def sync_volume_size(volume_uuid, session_uuid=None):
     action.uuid = volume_uuid
     try:
         evt = execute_action_with_session(action, session_uuid)
+        return evt.inventory
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)